test/Magick.rb

```diff
@@ -262,35 +262,35 @@ def test_limit_resources
 
     expect { cur = Magick.limit_resource(:memory, 500) }.not_to raise_error
     expect(cur).to be_kind_of(Integer)
-    assert(cur > 1024**2)
+    expect(cur > 1024**2).to be(true)
```

Comment:
Think it would be nicer to use `expect(cur).to be > 1024**2` here instead.

Reply:
@dlemstra I spent a little while trying to get this to work, but didn't have much luck. I'll take a note to switch it once RSpec is driving the tests.

Reply:
Okay 👍